modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/hooks/useDataLayouts.es.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import {getItem} from 'app-builder-web/js/utils/client.es';
+import {errorToast} from 'app-builder-web/js/utils/toast.es';
+import {useEffect, useState} from 'react';
+
+export default function useDataLayouts(dataLayoutIds = [], dataDefinitionId) {
```

Comment:
@naraandrade if you want to use defaultValues on functions arguments, you shouldn't use defaultValue just on the first argument if you have others without defaultValues. You could reorder them: function(dataDefinitionId, dataLayoutIds = []) { ... }, or use options argument as an object: function ({dataLayoutIds = [], dataDefinitionId}) {...}